bin/app.py

```diff
@@ -14,19 +14,15 @@
 
 # Load file
 # ----------------------------------------------------------------------------
-summary_report = "./outdir/summary_report/evaluation_summary_report.csv"
-stats_report = "./outdir/stats/stats_summary_report.csv"
+summary_report = "./shiny_data.csv"
```

Comment:
How good is shiny throwing exceptions when data is not present, maybe worth adding some informative message in case the data is not available?

Reply:
Yes!! I just added it :) Thanks for the suggestion!